Pyrado/pyrado/algorithms/step_based/value_based.py

```diff
@@ -44,7 +44,7 @@
 from pyrado.utils.input_output import print_cbt_once
 
 
-class ValueBased(Algorithm, ABC):
+class ValueBased(Algorithm, ExposedSampler, ABC):
```

Comment:
Why should this inherit from ExposedSampler?

Reply:
I thought that inserting `ExposedSampler` higher up the class hierarchy so that it is automatically inherited by all value-based algorithms like `SAC` or `DQL`. I can also move it down to the concrete subclasses.

Reply:
In general, I agree with your approach. However, the sublcasses have more than one sampler, and we did not find a solution fo that scenario, i.e. which sampler should be exposed, since none of them is called "sampler"

Reply:
It's removed in d1a2d9ce1e82f708d9717366d9811f8583b898a6